examples/treegrid/treegrid-1.html

```diff
@@ -370,30 +330,200 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
+              <li>In this implementation, the first <code>row</code> in the <code>treegrid</code> is included in the tab sequence when the page loads.</li>
+              <li>
+                When the user moves focus in the <code>treegrid</code>, the element included in the tab sequence changes to the element with focus as described in the section on
+                <a href="../../../#kbd_roving_tabindex">roving tabindex.</a>
+              </li>
+             </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-expanded=<q>false</q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            <ul>
+              <li>Indicates the <code>row</code> can show or hide other message rows within this <code>treegrid</code>.</li>
+              <li><code>aria-expanded=<q>false</q></code> indicates the message rows that responded to this e-mail are hidden.</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-expanded=<q>true</q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            <ul>
+              <li>Indicates the <code>row</code> can show or hide other message rows within this <code>treegrid</code>.</li>
+              <li><code>aria-expanded=<q>true</q></code> indicates the message rows that responded to this e-mail are visible.</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-hidden=<q>false</q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            <ul>
+              <li><code>aria-hidden</code> is used to control the information available to assisitve technologies.</li>
+              <li>When the <code>row</code> is visible (e.g. a top level message or a message whose parent message is expanded), <code>aria-hidden=<q>false</q></code> makes the message row information available to assistive technologies.</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-hidden=<q>true</q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            <ul>
+              <li><code>aria-hidden</code> is used to control the information available to assisitve technologies.</li>
+              <li>When a message <code>row</code> is <strong>not</strong> visible (e.g. is parent message is collapsed), <code>aria-hidden=<q>true</q></code> <strong>hides</strong> the message row  information from assistive technologies.</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-level=<q><em>number</em></q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            <ul>
```

Comment:
For simplicity and clarity, I recommend replacing the bullets in this list with the following to be consistent with the wording we use in the tree examples:

* Defines the level of the row in the hierarchical treegrid structure.
* Counting is one-based.
* Root rows have aria-level=“1”.